corehq/apps/accounting/migrations/0086_add_note_in_invoice_models.py

```diff
@@ -22,9 +22,7 @@ def _add_note_to_existing_invoice_duplicates(apps, schema_editor):
                                                         date_end=duplicate['date_end']).order_by('date_created')
             first_invoice = duplicate_invoices.first()
             for invoice in duplicate_invoices[1:]:
-                note = f"Invoice {invoice.id} is a duplicate of invoice id: {first_invoice.id}."
-                invoice.note = note
-                invoice.save()
+                invoice.update(note=f"Invoice {invoice.id} is a duplicate of invoice id: {first_invoice.id}.")
```

Comment:
Have you tested that this will work? Maybe I'm looking in the wrong place, but `Invoice` doesn't seem to have an `update` method. The method I suggested to use was [`Queryset.update`](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#update), but as you point out that will not work with the unique constraint.

Reply:
No I didn't. I just realize the method you suggested is different than what I used. And I think you're right, it won't work.